src/mmw/js/src/modeling/models.js

```diff
@@ -1003,8 +1005,18 @@ var ScenariosCollection = Backbone.Collection.extend({
             newModel = new ScenarioModel({
                 is_current_conditions: false,
                 name: this.makeNewScenarioName('Copy of ' + source.get('name')),
+                user_id: source.get('user_id'),
                 inputs: source.get('inputs').toJSON(),
-                modifications: source.get('modifications').toJSON()
+                inputmod_hash: source.get('inputmod_hash'),
+                modifications: source.get('modifications').toJSON(),
+                modification_hash: source.get('modification_hash'),
+                job_id: source.get('job_id'),
+                poll_error: source.get('poll_error'),
+                results: source.get('results').toJSON(),
+                has_fetched_results: source.get('has_fetched_results'),
```

Comment:
Oh no. This was used in the original implementation of fixing that empty `aoi_census` + `modifications` bug (the one that addressed if the initial model failed).  We can get rid of it entirely. @kellyi ...would you mind adding a commit to delete[ it from the model](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/modeling/models.js#L611)?